wrangling.Rmd

```diff
@@ -1625,39 +1625,36 @@ found in Chapter \@ref(move-to-your-own-machine).
-    another resource to learn more about the functions in this
-    chapter, the full set of arguments you can use, and other related functions.
-    The site also provides a very nice cheat sheet that summarizes many of the
-    data wrangling functions from this chapter.
-  - Check out the [`tidyselect` page](https://tidyselect.r-lib.org/reference/select_helpers.html) for a
-    comprehensive list of `select` helpers.
-  - [*R for Data Science*](https://r4ds.had.co.nz/) has a few chapters related to
-    data wrangling that go into more depth than this book. For example, the
-    [tidy data](https://r4ds.had.co.nz/tidy-data.html) chapter covers tidy data,
-    `pivot_longer`/`pivot_wider` and `separate`, but also covers missing values
-    and additional wrangling functions (like `unite`). The [data
-    transformation](https://r4ds.had.co.nz/transform.html) chapter covers
-    `select`, `filter`, `arrange`, `mutate`, and `summarize`. And the [`map`
-    functions](https://r4ds.had.co.nz/iteration.html#the-map-functions) chapter
-    provides more about the `map` functions.
-  - You will occasionally encounter a case where you need to iterate over items
-    in a data frame, but none of the above functions are flexible enough to do
-    what you want. In that case, you may consider using [a for
-    loop](https://r4ds.had.co.nz/iteration.html#iteration).
-  - There are many `select` helpers that can be used to efficiently subset 
-    columns in a data frame when paired with the `select` function, 
-    or other functions that also use the `tidyselect` syntax for column selection 
-    (e.g., `pivot-longer`). 
-    The [documentation for `select` helpers](https://tidyselect.r-lib.org/reference/select_helpers.html) 
-    is a useful reference to find the helper you need for your particular problem.
+- As we mentioned earlier, `tidyverse` is actually an *R
+  meta package*: it installs and loads a collection of R packages that all
+  follow the tidy data philosophy we discussed above. One of the `tidyverse`
+  packages is `dplyr`&mdash;a data wrangling workhorse. You have already met many
+  of `dplyr`'s functions 
+  (`select`, `filter`, `mutate`, `arrange`, `summarize`, and `group_by`). 
+  To learn more about these functions and meet a few more useful
+  functions, we recommend you check out Chapters 5-9 of the [STAT545 online notes](https://stat545.com/).
+  of the data wrangling, exploration, and analysis with R book.
+- The [`dplyr` R package documentation](https://dplyr.tidyverse.org/) [@dplyr] is
+  another resource to learn more about the functions in this
+  chapter, the full set of arguments you can use, and other related functions.
+  The site also provides a very nice cheat sheet that summarizes many of the
+  data wrangling functions from this chapter.
+- Check out the [`tidyselect` R package page](https://tidyselect.r-lib.org/reference/select_helpers.html) 
```

Comment:
Footnote 4 and 5 are the same link. I think 4 should be the tidyselect page and then 5 should be the select helpers page

Reply:
fixed!